### PR TITLE
Fix service declaration deprecation for api_platform.router

### PR DESCRIPTION
**Pull request by Zayon**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| License       | MIT

### CODE REVIEW

File: src/Bridge/Symfony/Bundle/Resources/config/api.xml [hunk 1]
```diff
@@ -44,6 +44,8 @@
             <tag name="routing.loader" />
         </service>
 
+        <service id="ApiPlatform\Core\Api\UrlGeneratorInterface" alias="api_platform.router" public="true" />
```
Comment (dunglas) [inline src/Bridge/Symfony/Bundle/Resources/config/api.xml]: `public="true"` should be removed (there is no reason to make this alias public).

**Comment by dunglas** [comment]
Can you rebase against 2.1 please? To me it's a bug fix.

**Comment by dunglas** [comment]
You also to add the alias in this array to fix the tests: https://github.com/api-platform/core/blob/master/tests/Bridge/Symfony/Bundle/DependencyInjection/ApiPlatformExtensionTest.php#L493

**Comment by Zayon** [comment]
ping @dunglas. I've updated the PR.

**Comment by dunglas** [comment]
Thanks @Zayon

**Pull request merged**